doc/api/https.md

```diff
@@ -124,150 +119,92 @@ https.get('https://encrypted.google.com/', (res) => {
 added: v0.5.9
 -->
 
-Global instance of [`https.Agent`][] for all HTTPS client requests.
+Global instance of [`https.Agent`] for all HTTPS client requests.
 
-## https.request(options, callback)
+## https.request(options[, callback])
 <!-- YAML
 added: v0.3.6
 -->
 
-Makes a request to a secure web server.
+* `options` {Object | String} Contains request and connection details. If `options`
+  is a string, it is automatically parsed with [`url.parse()`]. If `options` is
+  an object, valid properties include those for [`http.request()`], except
+  `protocol` defaults to `'https:'`. Additionally, options for [`tls.connect()`]
+  can be specified (however, the [`https.globalAgent`] silently ignores these).
+* `callback` {Function} If supplied, `callback` is automatically added as a
+  `response` event handler
+* Return: [`http.ClientRequest`] instance
 
-`options` can be an object or a string. If `options` is a string, it is
-automatically parsed with [`url.parse()`][].
-
-All options from [`http.request()`][] are valid.
+Creates and initiates a request to an HTTPS server.
 
 Example:
 
 ```js
 const https = require('https');
 
-var options = {
-  hostname: 'encrypted.google.com',
+const options = {
+  host: 'encrypted.google.com',
```

Comment:
`hostname` is just a result of calling out to `url.parse()`. Since this example is passing an object instead of a string, it doesn't really matter. I chose `host` because it's shorter.